src/core/core.tooltip.js

```diff
@@ -782,6 +782,11 @@ module.exports = function(Chart) {
 
 			// Remember Last Actives
 			changed = !helpers.arrayEquals(me._active, me._lastActive);
+
+			if (!changed) {
```

Comment:
Could you add a comment here explaining why this exists? It might be a good idea to include a link to the relevant github issue as well